src/test/java/seedu/duke/command/GetCommandTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.duke.command;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import seedu.duke.model.Item;
+import seedu.duke.model.ItemContainer;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class GetCommandTest {
+    private ItemContainer testList;
+    private Command testCommand;
+    private final PrintStream standardOut = System.out;
+    private final ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        testList = new ItemContainer("test");
+        testList.addItem(new Item("HarryPotter", "16.1", "25.12"));
+        testCommand = new GetCommand("HarryPotter", testList);
+        System.setOut(new PrintStream(outputStreamCaptor));
```

Comment:
Why is there a need to output something?

Reply:
To compare the output of print statement to expected output